fgcm/fgcmUtilities.py

```diff
@@ -471,11 +471,12 @@ def plotCCDMap(ax, deltaMapper, values, cbLabel, loHi=None):
     ax.scatter(deltaMapper['delta_ra'].ravel(), deltaMapper['delta_dec'].ravel(),
                s=0.1,
                c=zGrid.ravel(),
-               vmin=lo, vmax=hi)
+               vmin=lo, vmax=hi,
+               cmap=cm)
```

Comment:
You're not using `cNorm` or `scalarMap` from above anywhere. I suspect maybe `cNorm` was supposed to be used here instead of setting `vmin` and `vmax`?